lib/src/facts/openbsd/processor_resolver.cc

```diff
@@ -0,0 +1,67 @@
+#include <internal/facts/openbsd/processor_resolver.hpp>
+#include <leatherman/logging/logging.hpp>
+#include <sys/types.h>
+#include <sys/sysctl.h>
+
+using namespace std;
+
+namespace facter { namespace facts { namespace openbsd {
+
+    processor_resolver::data processor_resolver::collect_data(collection& facts)
+    {
+        auto result = posix::processor_resolver::collect_data(facts);
+        size_t len;
+        int mib[2];
+        mib[0] = CTL_HW;
+
+        // Get the physical count of processors
+        len = sizeof(result.logical_count);
+        mib[1] = HW_NCPUFOUND;
```

Comment:
Okay thanks for fixing the copy pasta. If it works, that's great. The naming/documentation for the constants may just be unintuitive to me. Thanks!